examples/CMakeLists.txt

```diff
@@ -284,8 +284,11 @@ if(LINUX)
             SRCS dram_and_fsdax/dram_and_fsdax.c
             LIBS umf jemalloc_pool)
```

Comment:
Do we still have `jemalloc_pool` library? Now jemalloc statically linked to the UMF right? So, should example still link with `jemalloc_pool`?

Reply:
Yes, we still have the static `jemalloc_pool` library.
No, jemalloc is statically linked to the `jemalloc_pool` library (not to the UMF).
Yes, the example should still link with the `jemalloc_pool` library, because it uses its API (`umfJemallocPoolOps` for example).

Reply:
Hmm, why did we make such a decision to keep the static `jemalloc_pool` library? Other pools are part of libumf. Why do we need a separate static `jemalloc_pool` library?

Reply:
OK, I understand now. I will fix it.

Reply:
No, removed.
Done.